packages/plugins/plugin-navtree/src/NavTreePlugin.tsx

```diff
@@ -130,18 +131,21 @@ export const NavTreePlugin = (): PluginDefinition<NavTreePluginProvides> => {
   return {
     meta,
     ready: async (plugins) => {
+      state.prop({ key: 'open', type: LocalStorageStore.json<string[]>() });
+
       graphPlugin = resolvePlugin(plugins, parseGraphPlugin);
       graph = graphPlugin?.provides.graph;
       if (!graph) {
         return;
       }
 
-      state.values.root = graph.root as NavTreeItemGraphNode;
-      getChildren(graph, state.values.root);
-      getActions(graph, state.values.root);
```

Comment:
these were old cruft, they used to have side-effects like "expand" call below but no longer do so they were doing nothing